packages/editor/src/client/app.tsx

```diff
@@ -24,6 +24,12 @@ import {PaymentMethodList} from './routes/paymentMethodList'
 import {NavigationList} from './routes/navigationList'
 
 import './global.less'
+import {IntlProvider} from 'rsuite'
+import enGB from 'rsuite/lib/IntlProvider/locales/en_GB'
+import fr from './locales/rsuiteFr'
+import en from './locales/rsuiteEn'
```

Comment:
I think the `rsuiteEn` is the same as the one you import on line 28 from the `.../lib/IntlProvider/locales/...` you can do it both way but i suggest to use the one provided by rsuite for the english language.